test/e2e/bots_test.go

```diff
@@ -1226,63 +1253,1151 @@ func runBotTest(t *testing.T,
-	err := ingressCli.Delete(context.Background(), name, metav1.DeleteOptions{})
-	assert.NoError(t, err)
-}
+	t.Log("Running actual test cases")
 
-func cleanupCreatedClusterRole(t *testing.T, clusterRoleCli rbacv1.ClusterRoleInterface, name string) {
-	t.Log("Cleaning up created ClusterRole...")
-	err := clusterRoleCli.Delete(context.Background(), name, metav1.DeleteOptions{})
-	assert.NoError(t, err)
-}
+	t.Run("Ping", func(t *testing.T) {
+		command := "ping"
+		expectedMessage := fmt.Sprintf("`%s` on `%s`\n```\npong", command, appCfg.ClusterName)
 
-func cleanupCreatedClusterRoleBinding(t *testing.T, clusterRoleBindingCli rbacv1.ClusterRoleBindingInterface, name string) {
-	t.Log("Cleaning up created ClusterRoleBinding...")
-	err := clusterRoleBindingCli.Delete(context.Background(), name, metav1.DeleteOptions{})
-	assert.NoError(t, err)
+		botDriver.PostMessageToBot(t, botDriver.Channel().Identifier(), command)
+		err := botDriver.WaitForLastMessageContains(botDriver.BotUserID(), botDriver.Channel().ID(), expectedMessage)
+		assert.NoError(t, err)
+	})
+
+	t.Run("Help", func(t *testing.T) {
+		command := "help"
+		expectedMessage := interactive.NewHelpMessage(config.CommPlatformIntegration(botDriver.Type()), appCfg.ClusterName, []string{"botkube/helm", "botkube/kubectl"}).Build()
+		expectedMessage.ReplaceBotNamePlaceholder(botDriver.BotName(), api.BotNameWithClusterName(appCfg.ClusterName))
+		botDriver.PostMessageToBot(t, botDriver.Channel().Identifier(), command)
+		err = botDriver.WaitForLastInteractiveMessagePostedEqual(botDriver.BotUserID(),
+			botDriver.Channel().ID(),
+			expectedMessage,
+		)
+
+		assert.NoError(t, err)
+	})
+
+	t.Run("Botkube PluginManagement", func(t *testing.T) {
+		t.Run("Helm Executor", func(t *testing.T) {
+			command := "helm install --help"
+			expectedBody := codeBlock(heredoc.Doc(`
+					Installs a chart archive.
+
+					There are two different ways you to install a Helm chart:
+					1. By absolute URL: helm install mynginx https://example.com/charts/nginx-1.2.3.tgz
+					2. By chart reference and repo url: helm install --repo https://example.com/charts/ mynginx nginx
+
+					Usage:
+					    helm install [NAME] [CHART] [flags]
+
+					Flags:
+					    --create-namespace
+					    --generate-name,-g
+					    --dependency-update
+					    --description
+					    --devel
+					    --disable-openapi-validation
+					    --dry-run
+					    --insecure-skip-tls-verify
+					    --name-template
+					    --no-hooks
+					    --pass-credentials
+					    --password
+					    --post-renderer
+					    --post-renderer-args
+					    --render-subchart-notes
+					    --replace
+					    --repo
+					    --set
+					    --set-json
+					    --set-string
+					    --skip-crds
+					    --timeout
+					    --username
+					    --verify
+					    --version
+					    -o,--output`))
+			expectedMessage := fmt.Sprintf("%s %s, with interactive elements", cmdHeader(command), expectedBody)
```

Comment:
> "with interactive elements"

how it's produced? I can't see this in our code base 🤔

Reply:
Actually, we are using sPrintedBlocks, there this `with interactive elements` comes from Slack. Slack puts this message instead of putting interactive elements' content

Reply:
Moved this logic to internal slack fmt functions so that this text will be removed like we normalize links in slack messages